engagement_database/engagement_database.py

```diff
@@ -49,22 +49,22 @@ def _messages_ref(self):
     def _message_ref(self, message_id):
         return self._messages_ref().document(message_id)
 
-    def get_history_for_message(self, message_id, filter=lambda q: q, transaction=None):
+    def get_history_for_message(self, message_id, query_filter=lambda q: q, transaction=None):
```

Comment:
I wonder if we ought to call this something like firestore_filter, or firestore_query_filter, since this is exposing the underlying firestore query rather than providing a database-agnostic wrapper for it.

Reply:
I think `firestore_query_filter` makes sense because the filter function returns a query.

Reply:
agreed (sorry to make you change it again!)

Reply:
No problem.

Reply:
`firestore_query_filter` is more descriptive. Thanks.

Reply:
Updated, thanks.